### PR TITLE
define mac.allsig etc. when enrich=FALSE

### PR DESCRIPTION
**Pull request by bpolacco**
Crash at line number 1928 (was 1925) when these are not defined in case enrich=FALSE

### CODE REVIEW

File: R/analysisQuantifications.R [hunk 1]
```diff
@@ -1403,6 +1403,9 @@ artmsAnalysisQuantifications <- function(log2fc_file,
     }
   } else{
     if(verbose) message(">> NO ENRICHMENT of CHANGES (log2fc) SELECTED ")
+    mac.allsig <- NULL
```
Comment (biodavidjm) [inline R/analysisQuantifications.R:1406]: @bpolacco why is this necessary? would you mind explaining it a little bit? Thanks!

Comment (bpolacco) [reply R/analysisQuantifications.R:1406]: When enrich=FALSE,  I get a crash at line 1928 (was 1925) with an error `Error: object mac.allsig not found`.   The variable is only defined in the block of code  in if(enrich==TRUE), so it needs to be defined somewhere before you test `is.null(mac.allsig)` etc at line 1928.

Comment (biodavidjm) [reply R/analysisQuantifications.R:1406]: Great, thanks. I'll merge it tonight and push to Bioconductor. 
Next time, I would suggest creating an issue, explain a little bit what the error is (with the error output) and then when doing the pull request, reference the issue. In that way is easier to understand what the PR is all about. 
Again, thanks a lot for your contribution!

Comment (bpolacco) [reply R/analysisQuantifications.R:1406]: Will try to communicate better.  Time pressure makes that difficult usually though.  Thanks!
